### ISSUE TITLE
RFC: What would you like to see in Keywind?

### ISSUE THREAD

**Issue by lukin** [lukin/keywind#6]
Hey 👋

I need your help! I would really love to hear what you would like to see in Keywind.

Features:

- [ ] [Dark Mode](https://tailwindcss.com/docs/dark-mode)
- [ ] Unit Testing

Pages:
- [ ] [Login Verify Email](https://github.com/keycloak/keycloak/blob/main/themes/src/main/resources/theme/base/login/login-verify-email.ftl)

**Comment by magick93** [comment]
CI/CD pipeline would be great. Something that can package up the theme into an archive, then add it to github releases.
Then it can easily be installed using the Keycloak Operator.

And yes for unit testing :)

**Comment by deedubs** [comment]
I would definitely appreciate releases.  I'm not sure if there is a security concern around using a pre-package theme 😨

**Comment by mnaser** [comment]
> I would definitely appreciate releases. I'm not sure if there is a security concern around using a pre-package theme 😨

yeah, it'd be nice, it's taking me quite a bit of time to be able to get it working :)

**Comment by fbrettnich** [comment]
Identity Provider Icon for Discord (https://github.com/wadahiro/keycloak-discord)

**Comment by kaleabmelkie** [comment]
Better guides/docs on how to set it up

**Comment by getakalabs-christarroja** [comment]
account console page

**Comment by hbadri3** [comment]
RTL please

**Comment by AlmondMilk4242** [comment]
Ill actually like someone to setup a development place that easily enables user to load that code into the ftl. its time consuming to modify it in ftl. and with dev stage in place, modifications and scaling pages are easy

**Comment by tomas-rojo** [comment]
I would love to see the account console.

**Comment by abhishekgahlot2** [comment]
JSX Components please

**Comment by CHE1RON** [comment]
Email theme! (duplicate of #47)

**Comment by manuschillerdev** [comment]
provide the theme as docker image that can be loaded as sidecar

**Comment by ByteAfterlife** [comment]
I'd love dark mode!
